elastic/apm/index-template.json

```diff
@@ -0,0 +1,9 @@
+{
+    "index_patterns": ["traces-apm*", "logs-apm*", "metrics-apm*"],
+    "data_stream": {},
+    "_meta": {
+        "description": "Placeholder template that'd be used when the x-pack-apm car is not used, e.g. in test mode"
+    },
+    "composed_of": ["apm@settings"],
```

Comment:
Using `apm@settings` is a bit brittle. Yes the apm-data plugin does use it, but it's not intended as an extension point -- it's a built-in component template that is not intended to be modified by users.

I would strongly suggest using `traces@custom`, `logs@custom`, and `metrics@custom` instead.